documentation/specs/BuildCheck/Codes.md

```diff
@@ -121,7 +121,7 @@ If `BC0202` and [BC0201](#BC0201) are both enabled - then `BC0201` reports only
 
 This check indicates that a property was defined in the observed scope (by default it's the project file only) and it was then not used anywhere in the build.
 
-This is a runtime check, not a static analysis check - so it can have false positives - for this reasons it's currently only suggestion.
+This is a runtime check, not a static analysis check - so it can have false positives - for this reasons it's currently not enabled by defaut.
```

Comment:
Could we add some instructions on how to enable it if the user wants the check?

Reply:
I've added general note on how to reconfigure the default config: https://github.com/dotnet/msbuild/pull/11055/files#diff-e96bdd5e98d7e8729178bfbea183e2e16e385f10219a0d4121b2043a4883d5d9R20